analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/converter/ProjectConverter.kt

```diff
@@ -42,12 +42,13 @@ class ProjectConverter(private val containsAuthors: Boolean) {
         return edge
     }
 
-    fun convert(versionControlledFiles: List<VersionControlledFile>, metricsFactory: MetricsFactory): Project {
+    fun convert(versionControlledFiles: MutableMap<String, VersionControlledFile>?, metricsFactory: MetricsFactory): Project {
```

Comment:
I guess the `versionControlledFiles` is always present?

```suggestion
    fun convert(versionControlledFiles: MutableMap<String, VersionControlledFile>, metricsFactory: MetricsFactory): Project {
```